README.md

```diff
@@ -153,9 +153,88 @@ This library doesn't handle this for you, but here are the 2 main options:
 This is the easiest method: by rendering your form inside a live component,
 it will automatically re-render when the form changes.
 
-### B) Write custom JavaScript
+### B) Use [Symfony UX Turbo](https://symfony.com/bundles/ux-turbo/current/index.html#decomposing-complex-pages-with-turbo-frames)
+
+If you are already using Symfony UX Turbo on your website, you can have a dynamic form running quickly without any JavaScript.
+
+Or you may want to install Symfony UX Turbo, [check out the documentation](https://symfony.com/bundles/ux-turbo/current/index.html#installation).
+
+> [!NOTE]
+> You only need to have Turbo Frame, you can disable Turbo Drive if you do not use it, or do not want to use it.
+> ie: `Turbo.session.drive = false;`
+
+Simply add a `<turbo-frame>` around your form:
+
+```twig
+<turbo-frame id="rating-form">
+    {{ form(form) }}
+</turbo-frame>
+```
+
+From here you need two small changes:
+
+First, in your form type:
+
+```php
+// src/Form/FeedbackForm.php
+
+// ...
+
+    $builder->add('rating', ChoiceType::class, [
+        'choices' => [
+            'Select a rating' => null,
+            'Great' => 5,
+            'Good' => 4,
+            'Okay' => 3,
+            'Bad' => 2,
+            'Terrible' => 1
+        ],
+        // This will allow the form to auto-submit on value change
+        'attr' => ['onchange' => 'this.form.submit()'],
+    ]);
+    // This will allow to differenciate between a user submition and an auto-submit
+    $builder->add('submit', SubmitType::class, [
+        'attr' => ['value' => 'submit'], // Needed for turbo
+    ]);
+
+```
+
+Second, in your controller:
```

Comment:
Same here, I would like to mention what exactly devs should do in their controllers and why. And `diff` code block instead of PHP may help to notice the change easier. Thoughts?

Reply:
Agreed for more detail textual description of the changes, but for the `diff` format, if used, we lose the `php` highlighting, and it's kind of hard to read then (or maybe we can use both? diff+php).

Reply:
Hm, not sure `diff+php` will work, though never tried it. Yeah, with diff will lose PHP highlighting, but if we're talking about a few new lines that should be added in a big code block - I'm fine with it. It would help to focus on specific lines